server/models/article.js

```diff
@@ -20,7 +39,7 @@ const articleSchema = new Schema({
     unique: true,
     required: true,
   },
-  title: { type: String, trim: true, required: true },
+  info: mercurySchema,
```

Comment:
can you explain this line? any reason why its not an object id referencing mercurySchema?